packages/beacon-node/test/e2e/doppelganger/doppelganger.test.ts

```diff
@@ -90,11 +91,12 @@ describe.skip("doppelganger / doppelganger test", function () {
     const pubKey = validatorUnderTest.validatorStore.votingPubkeys()[0];
     const beaconBlock = ssz.allForks.phase0.BeaconBlock.defaultValue();
 
+    // Signing should be possible if starting at genesis since doppelganger should be off
     await expect(
-      validatorUnderTest.validatorStore.signBlock(fromHexString(pubKey), beaconBlock, bn.chain.clock.currentSlot),
-      "Signing should be possible if starting at genesis since doppelganger should be off"
-    ).to.eventually.be.fulfilled;
+      validatorUnderTest.validatorStore.signBlock(fromHexString(pubKey), beaconBlock, bn.chain.clock.currentSlot)
+    ).resolves.toBeUndefined();
```

Comment:
if this assertion fails now the error message will be pretty meaningless, can we use withMessage here?

Reply:
yes we can that was making the assertion looks odd. Updated it with `toBeWithMessage` now.